tests/drivers/i2c/i2c_target_api/boards/frdm_mcxn236.overlay

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
need change: testcase.yaml adding this platform to:
`drivers.i2c.target_api.single_role:`